db/schema_tables.cc

```diff
@@ -1096,14 +1131,18 @@ static future<> do_merge_schema(distributed<service::storage_proxy>& proxy, std:
     auto&& new_views = co_await read_tables_for_keyspaces(proxy, keyspaces, views());
     auto new_functions = co_await read_schema_for_keyspaces(proxy, FUNCTIONS, keyspaces);
     auto new_aggregates = co_await read_schema_for_keyspaces(proxy, AGGREGATES, keyspaces);
+    auto new_scylla_aggregates = schema_result();
+    if (proxy.local().features().cluster_schema_features().contains<schema_feature::SCYLLA_KEYSPACES>()) {
+        new_scylla_aggregates = co_await read_schema_for_keyspaces(proxy, SCYLLA_AGGREGATES, keyspaces);
+    }
```

Comment:
same here.

Reply:
but you should check the feature when exporting schema mutations to send to another node.